### DISCUSSION TITLE
Not selected image when upload can tiptap do this?

### DISCUSSION THREAD

**Discussion by patsornk** [ueberdosis/tiptap#1802] [General Questions]
when upload image. Cursor selected that image but I don't want it. Can I?

![image](https://user-images.githubusercontent.com/76081944/130913770-6c24e9ce-1aad-42c9-918a-e5ad6d990022.png)

**Comment by hanspagel** [comment]
This is CSS. Look for `ProseMirror-selectednode` in your CSS and remove it.

**Reply by patsornk** [reply]
But It still selected I mean when I add 2 image 2nd image was deleted
![image](https://user-images.githubusercontent.com/76081944/131099016-1cf775ac-0208-4a5e-9231-15fbbc3a5d06.png)

**Reply by cgorrieri** [reply]
Hello, it seems that the plugin doesn't select the image automatically after inserting it. I would like to enable this behavior, how can I do that?

**Reply by YassineBouzeya** [reply]
Hi guys, you can set your image plugin to enable the "inline option". This way, when you upload your image it will not be selected by default.

https://tiptap.dev/api/nodes/image#inline

```
Image.configure({
  inline: true,
})
```
